### ISSUE TITLE
misinterpretation and crash when using normal date format

### ISSUE THREAD

**Issue by asenchristov** [google/ezgantt#10]
When using the normal date format (i.e. dd/mm/yyyy), it assumes the strange US format where the month is the first number.

**Comment by salamanders** [comment]
True, it isn't a pretty way to parse dates that respects local timezones: https://github.com/google/ezgantt/blob/master/website/js/ezgantt.js#L146

Open to suggestions!

**Comment by salamanders** [comment]
I was strongly considering moving to moment.js and turning on strict mode parsing.
